picturae_importer.py

```diff
@@ -8,18 +8,16 @@
 import logging
 from string_utils import *
 from importer import Importer
-from picturae_csv_create import CsvCreatePicturae
 from sql_csv_utils import SqlCsvTools
 from botany_importer import BotanyImporter
-from picturae_csv_create import starting_time_stamp
-from specify_db import SpecifyDb
 from get_configs import get_config
 from os import path
 import time_utils
 import logging.handlers
 from monitoring_tools_derived import MonitoringToolsDir
 from taxon_tools.BOT_TNRS import process_taxon_resolve
 
```

Comment:
moved starting timestamp to beginning of importer rather than picturae_csv_create